graphene/relay/connection.py

```diff
@@ -25,10 +25,10 @@ class EdgeMeta:
         description = f"A Relay edge containing a `{base_name}` and its cursor."
 
     edge_name = f"{base_name}Edge"
-    if edge_class:
-        edge_bases = [edge_class, EdgeBase, ObjectType]
-    else:
-        edge_bases = [EdgeBase, ObjectType]
+
+    edge_bases = [edge_class] if edge_class else [EdgeBase]
```

Comment:
My bad in not being specific with my comment 🙂  If EdgeBase doesn't inherit from `ObjectType`. This should of course be `edge_bases = [edge_class] if edge_class else [EdgeBase, ObjectType]` 🙂

Reply:
Should be resolved now, right? The next line practically does that for both.

Reply:
Ohh sure 🙈